pandas/tests/tools/test_to_timedelta.py

```diff
@@ -24,6 +24,18 @@ def test_to_timedelta(self):
         result = to_timedelta(Series(["1d", "1days 00:00:01"]))
         tm.assert_series_equal(result, expected)
 
+        # IntegerArray Series
+        expected = Series([timedelta(days=1), timedelta(days=2)])
+        result = to_timedelta(Series([1, 2], dtype="Int64"), unit="days")
+
+        tm.assert_series_equal(result, expected)
+
+        # IntegerArray Series with nulls
```

Comment:
it looks like other tests for missing values are in 
https://github.com/pandas-dev/pandas/blob/3701a9b1bfc3ad3890c2fb1fe1974a4768f6d5f8/pandas/tests/tools/test_to_timedelta.py#L133
so might want to move this part there

Reply:
Alternatively, I would maybe just put this new code in its own test (so a test specifically for passing nullable Int64 dtype data)